docs/platforms/apple/common/overhead/index.mdx

```diff
@@ -0,0 +1,59 @@
+
+## Screenshots and View Hierarchy
+
+If you activate these features, the SDK will capture <PlatformLink to="/enriching-events/screenshots">Screenshots</PlatformLink> and <PlatformLink to="/enriching-events/viewhierarchy">View Hierarchy</PlatformLink> of the app's UI at the time of the error crash. This incurs a small overhead that is unnoticeable during normal operation.
+
+If your app raises many errors in a tight loop, it can become too much to process quickly enough, and UI jank can be the result, so make sure you handle such cases appropriately. The SDK provides callbacks to fine-tune when to capture <PlatformLink to="/enriching-events/screenshots#customize-screenshot-capturing">Screenshots</PlatformLink> and <PlatformLink to="/enriching-events/viewhierarchy#customize-view-hierarchy-capturing">View Hierarchy</PlatformLink>, which can also be used to reduce performance impact in scenarios where you don't need screenshots or view hierarchies.
+
+</PlatformSection>
+
+## Breadcrumbs
+
+<PlatformLink to="/enriching-events/breadcrumbs">Breadcrumbs</PlatformLink> are collected through automated integrations or by manually adding them. To have them readily available for every event generated by the SDK, they are continuously persisted, and managed in a performant buffer. This shouldn't impact user experience.
+
+Capturing excessive numbers of breadcrumbs (for example, creating breadcrumbs for all log messages) can cause significant performance overhead. To mitigate this, review and adapt your app's usage of breadcrumbs. For example, increase the min-level of log messages that create breadcrumbs from `warn` to `error`.
+
+Note that increasing the max number of breadcrumbs **does not** improve performance and can even have a detrimental effect.
+
+## Tracing and Performance Monitoring
+
+As stated in our <Link to="/product/performance/performance-overhead">product docs on the topic</Link>, Tracing adds some overhead, but should have minimal impact on the performance of your application. In typical scenarios, the expected overhead is less than 3% of the app's resource utilization.
+
+<PlatformSection supported={["apple.ios", "apple.macos"]}>
+
+## Profiling
+
+As stated in our <Link to="/product/explore/profiling/performance-overhead">product docs on the topic</Link>, Profiling adds some overhead, but should have minimal impact on the performance of your application. In typical scenarios, the expected overhead is less than 5% of the app's resource utilization.
+
+</PlatformSection>
+
+<PlatformSection supported={["apple.ios"]}>
+
+## Session Replay
+
+As stated in our <Link to="/product/explore/session-replay/mobile/performance-overhead">product docs on the topic</Link>, Session Replay adds some overhead, but should have minimal impact on the performance of your application. For more details on the measured overhead, read the <PlatformLink to="/session-replay/performance-overhead/">performance overhead docs for this SDK</PlatformLink>.
```

Comment:
Should we mention that every time we capture a screenshot you get a slightly delayed slow frame? Having it turned on causes plenty of slow frames for your UIViewController transactions, for example.

Reply:
I think this is mainly true for low end iOS devices, right? or in general?

might be something to (also) put on the SR perf overhead docs for SR. @brustolin @romtsn wdyt?

Reply:
Yes, I think it should be mentioned.

Reply:
I mean we kind of put the numbers there that imply that you'll have a slow frame, but yeah we could mention that explicitly:
<img width="754" alt="image" src="https://github.com/user-attachments/assets/ee9edd95-f1b7-407f-8960-c179814004d9">

Reply:
added a note below -> https://github.com/getsentry/sentry-docs/pull/12063/files#diff-47891e19e6939b70d674d775a7d082e3188f455a519dbab3f3cee073b35e7977R59